recipe/build.sh

```diff
@@ -23,7 +23,7 @@ fi
 export CLASSPATH=${PREFIX}/share/java/Jive.jar
 
 java \
-    -mx128m \
+    -mx1g \
```

Comment:
We should probably just remove this setting and keep the default.
If we don't pass it, we can as well use env variable to change the value.

This comes from https://gitlab.com/tango-controls/TangoSourceDistribution/-/blob/main/assets/lib/java/jive.in?ref_type=heads#L24

We should open an issue there and do the same thing.

Reply:
updated

Reply:
Thanks. Forgot to mention you have to update `jive.bat` as well for windows.

Reply:
done